### PR TITLE
Capitalize the word "Agent"

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Similar to #19043 

This was trickier, because it was in a lot more places, but also needed to read every change to make sure we are talking about our Agent.

### CODE REVIEW

File: src/web/api/netdata-swagger.yaml
Comment (ilyam8) [inline src/web/api/netdata-swagger.yaml:1]: Please revert all changes made to this file. One file needs to be manually updated, and then the other file needs to be regenerated based on the updated version.

Comment (Ancairon) [reply src/web/api/netdata-swagger.yaml:1]: Asked Stelios, we update yaml and generate json

File: src/web/api/netdata-swagger.yaml [hunk 1]
```diff
@@ -2561,33 +2561,33 @@ components:
     api:
       description: The version of the API used.
       type: integer
-    agents:
+    Agents:
```
Comment (ilyam8) [inline src/web/api/netdata-swagger.yaml:2564]: This looks wrong. I suggest skipping blind update of swagget files.

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: src/web/api/netdata-swagger.yaml (hunk 1 above)
Comment (Ancairon) [reply src/web/api/netdata-swagger.yaml:2564]: hey, true, it was not blind, I just missed it in the review process. I will look for such changes

**Pull request merged**